docs/visual-testing/integrations/csharp.md

```diff
@@ -333,6 +334,46 @@ VisualClient = VisualClient.Create(Driver, Region.UsWest1, sauceUsername, sauceA
 VisualClient.CaptureDom = true;
 ```
 
+### Full page screenshots
```

Comment:
There is no way to enable this option for every VisualCheck via Client options?

Reply:
no but we have task to do it
https://saucedev.atlassian.net/browse/IRIS-892